### PR TITLE
fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
fixes user and non-user facing typos. Found via `codespell -q 3 -S "./common/resources/lang/*,*.xml,*.properties" -L autor,beatin,componentes`

### CODE REVIEW

File: desktop/TuxGuitar/share/help/detail_file_edit.html [hunk 1]
```diff
@@ -94,7 +94,7 @@ <h2 id="edit_mode">Edit Mode (applies only to Score Editing)</h2>
     <img src="images/edit_mode.png" />
     <p>If your score staff is not visible, you can enable it in the <strong><a href="detail_view.html#layout">View</a></strong> menu.</p>
     <h3 id="selection_mode">Selection Mode</h3>
-    <p>The menu item <strong>Edit → Selection Mode</strong> (or the third icon in the <strong>Edit</strong> section of the Edit Toolbar) will put you into "Selection Mode". In this mode, you can click your mouse anywhere in the score staff, and it will simply move your playback cursor to that location, without making any changes to the score itself. Good for quickly moving around in your score without worrying about accidently altering notes.</p>
+    <p>The menu item <strong>Edit → Selection Mode</strong> (or the third icon in the <strong>Edit</strong> section of the Edit Toolbar) will put you into "Selection Mode". In this mode, you can click your mouse anywhere in the score staff, and it will simply move your playback cursor to that location, without making any changes to the score itself. Good for quickly moving around in your score without worrying about accidentallyly altering notes.</p>
```
Comment (guiv42) [inline desktop/TuxGuitar/share/help/detail_file_edit.html:97]: accidentallyly -> accidentally

File: desktop/TuxGuitar/share/help/edit_chord.html [hunk 2]
```diff
@@ -67,7 +67,7 @@ <h2 id="chord">Chord</h2>
     <p>Below the <strong style="color: #db291f">Chord</strong> controls there is a bass note. If you choose a different bass note than the chord root you will get so-called inverted chord.</p>
     <h2 id="proposals">Proposals</h2>
     <p>Based on the chosen chord theory, you will get proposals for the fingerings. On the <strong style="color: #db291f">Proposals</strong> thumbnails you can see the chord position and the fingering. The chord proposals are generated and sorted, so don’t be too disappointed if you don’t see your favorite chord on the second position.</p>
-    <p>On the other hand, proposals are generated for any tunning and any criteria the same way. You may be surprised when it digs some convenient fingering for a chord you had trouble with.</p>
+    <p>On the other hand, proposals are generated for any tuningand any criteria the same way. You may be surprised when it digs some convenient fingering for a chord you had trouble with.</p>
```
Comment (guiv42) [inline desktop/TuxGuitar/share/help/edit_chord.html:70]: tuningand -> tuning and

File: desktop/TuxGuitar/share/skins/blue_serious/skin.svg
Comment (guiv42) [inline desktop/TuxGuitar/share/skins/blue_serious/skin.svg:1]: Why a modification of this file?

File: desktop/TuxGuitar/share/skins/ersplus/skin.svg
Comment (guiv42) [inline desktop/TuxGuitar/share/skins/ersplus/skin.svg:1]: Why a modification of this file?

File: desktop/TuxGuitar/src/org/herac/tuxguitar/app/view/dialog/chord/TGChordCreatorUtil.java [hunk 3]
```diff
@@ -804,7 +804,7 @@ private float combinationHasAllRequiredNotes(List<StringValue> stringValueCombin
 		// required notes count should decrease the penalty
 		int noteCount = (this.alteration == 0 ? 0 : 1+ this.alteration)+currentIndex+ (this.bassTonic == this.chordTonic ? 0 : 1);
 		
-		// sometimes, when noteCount is bigger then tunning length, this pennalty will become positive, which may help
+		// sometimes, when noteCount is bigger then tuning length, this pennalty will become positive, which may help
```
Comment (guiv42) [inline desktop/TuxGuitar/src/org/herac/tuxguitar/app/view/dialog/chord/TGChordCreatorUtil.java:807]: not linked to the pull request, but can be corrected also: pennalty -> penalty

File: website/files/1.6.0/desktop/help/detail_composition_track.html
Comment (guiv42) [inline website/files/1.6.0/desktop/help/detail_composition_track.html:1]: This remark concerns all files modified in folders "website/files/1.6.x": these are archive folders, with the help files of previous application versions.
So they should not be modified (don't modify history, just fix the current version)

**Review by guiv42 — COMMENTED** [review]
Thanks for this PR. See my different remarks:
- a few minor remarks about typos
- one question: why are some svg files modified?
- do not modify history (website)

File: desktop/TuxGuitar/share/skins/blue_serious/skin.svg
Comment (luzpaz) [reply desktop/TuxGuitar/share/skins/blue_serious/skin.svg:1]: If you look a the SVG code there is a typo in one of the fields that it fixes. Same goes for the subsequent one

**Comment by helge17** [comment]
Thanks for you findings and comments, @luzpaz and @guiv42. I added the changes manually with commit b34c4838da797b9423aadaec38ac8ed3908419ba since it was to complicated to modify the PR. codespell finds even more typos, I will fix them in the next days.

**Comment by helge17** [comment]
Most codespell findings should be resolved now. If you find more, please open a new PR.

**Pull request closed without merging**